app/views/projects/new.html.erb

```diff
@@ -10,6 +10,7 @@
 
   <%= form.govuk_text_field :urn, label: { tag: 'h3', size: 'm' } %>
   <%= form.govuk_text_field :trust_ukprn, label: { tag: 'h3', size: 'm' } %>
+  <%= form.govuk_date_field :target_completion_date, omit_day: true %>
```

Comment:
That was easy! 👍

Reply:
It seems to render the day as the first by default? Is that documentend on the form builder?

Reply:
It is yep 

```
When mapped to a Date attribute, Ruby will automatically set the Day to be 1 when none is provided. This should be taken into account when displaying the data or using it for analysis.
```